core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -937,6 +937,28 @@ abstract class Record(private[chisel3] implicit val compileOptions: CompileOptio
     case _ => false
   }
 
+  trait AutoCloneType { self: Record =>
+    override def cloneType: this.type = _cloneTypeImpl.asInstanceOf[this.type]
+  }
+
+  private def checkClone(clone: Record): Unit = {
+    for ((name, field) <- elements) {
+      if (clone.elements(name) eq field) {
+        throw new AutoClonetypeException(
+          s"The bundle plugin was unable to clone $clone that has field '$name' aliased with base $this." +
+            " Try wrapping the field(s) in Input(...), Output(...), or Flipped(...) if appropriate." +
+            " As a last resort, you can override cloneType manually."
+        )
```

Comment:
This error message needs to be updated since users cannot override cloneType manually.

It also needs to give the user some hint for the case when AutoCloneType does not realize there are Data in the arguments to the Bundle or Record which in 3.5.6 gives rebinding exceptions but should be caught by this error. eg. see https://scastie.scala-lang.org/uMPihS2TSxy3sPCYfUb7AA for an example.

Reply:
You have a test below for this case, but this error needs to help the user figure out what went wrong.